rubyparser.h

```diff
@@ -130,79 +131,13 @@ enum node_type {
     NODE_HSHPTN,
     NODE_FNDPTN,
     NODE_ERROR,
+    NODE_DEF_TEMP,
+    NODE_DEF_TEMP2,
+    NODE_RIPPER,
+    NODE_RIPPER2,
```

Comment:
I think we should use better name...

Reply:
What do feel about `NODE_RIPPER_VALUES` instead of `NODE_RIPPER2`? VALUES comes from the fact all 3 fields are VALUE.